structuredefinitions/Extension-UKCore-DeliveryChannel.xml

```diff
@@ -1,74 +1,64 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-	<id value="Extension-UKCore-DeliveryChannel"/>
-	<url value="https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-DeliveryChannel"/>
-	<version value="1.2.0"/>
-	<name value="ExtensionUKCoreDeliveryChannel"/>
-	<title value="Extension UK Core Delivery Channel"/>
-	<status value="active"/>
-	<date value="2022-12-16"/>
```

Comment:
Does anyone know why the develop version had an updated date/version, content looks identical at first glance